### DISCUSSION TITLE
Problem with links

### DISCUSSION THREAD

**Discussion by oksiniichan** [nvim-neorg/neorg#1576] [Syntax]
![image](https://github.com/user-attachments/assets/6e36ec81-eb2b-4c39-b2d0-bfef26d43d32)

Doesn't seem to be hiding links. Becomes a problem when editing long names with long url links.

**Comment by max397574** [comment]
:h conceallevel

**Reply by oksiniichan** [reply]
what exactly do I have to search for?

**Reply by max397574** [reply]
this option really likely is what you haven't set properly

**Reply by oksiniichan** [reply]
Fixed.
Thanks for the help :D
